PorticoExportDom.php

```diff
@@ -35,61 +39,55 @@ class PorticoExportDom
     /** @var string DTD ID of the exported XML */
     private const PUBMED_DTD_ID = '-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.2 20190208//EN';
 
-    /** @var Context Context */
-    private $_context;
-
-    /** @var Issue Issue */
-    private $_issue;
-
-    /** @var Submission Submission */
-    private $_article;
-
-    /** @var DOMElement Document node */
-    private $_document;
-
-    /** @var Section Section */
-    private $_section;
+    private Context $context;
+    private DOMElement|DOMDocument $document;
+    private Issue $issue;
+    private Section $section;
+    private Submission $article;
 
     /**
      * Constructor
      */
     public function __construct(Journal $context, Issue $issue, Submission $article)
     {
-        $this->_context = $context;
-        $this->_issue = $issue;
-        $this->_article = $article;
-        if ($sectionId = $this->_article->getSectionId()) {
-            $this->_section = Repo::section()->get($sectionId);
+        $this->context = $context;
+        $this->issue = $issue;
+        $this->article = $article;
```

Comment:
Here you can simplify the construction

```suggestion
    private DOMElement|DOMDocument $document;
    private Section $section;

    /**
     * Constructor
     */
    public function __construct(private Journal $context, private Issue $issue, private Submission $article)
    {
```